src/httpHandlers/post/upload.ts

```diff
@@ -26,59 +24,36 @@ export const uploadPostHandler = async (req: Request, res: Response) => {
       });
     }
 
-    const fileName = req.body.fileName;
+    const objectKey = Date.now().toString() + req.body.fileName;
     const mimeType = req.body.mimeType;
-    const bytes = req.body.bytes;
+    const bytes = Buffer.from(req.body.bytes.replace(/^data:image\/\w+;base64,/, ""), "base64");
 
-    const saveResult = await saveImageToS3(fileName, bytes, mimeType);
+    let storage = new Storage({
+      credentials: Environment.googleCloudStorageCredentials
+    });
+
+    const uploadResult = await storage.bucket("local-api-server-ztq5-asia-southeast2-bucket").file(objectKey).save(bytes, {
```

Comment:
My suggestion would be to not hard code an eventual bucket here, but make that configurable via `GCS_BUCKET` or alike.